### PR TITLE
Correct tests for native functions' toString value

### PR DESCRIPTION
**Pull request by jugglinmike**
Modify the regular expression for native functions' toString value to
satisfy all test cases. Correct the test file's reference to the harness
file. Re-format the test file's assertions to aid debugging in the event
of failure.

@michaelficarra Would you mind taking a look?

### CODE REVIEW

File: harness/nativeFunctionMatcher.js [hunk 1]
```diff
@@ -1 +1 @@
-const NATIVE_FUNCTION_RE = /\bfunction\b[\s\S]+\b\w+\b[\s\S]*\([\s\S]*\)[\s\S]*\{[\s\S]*\[[\s\S]*\bnative\b[\s\S]+\bcode\b[\s\S]*\][\s\S]*\}/;
+const NATIVE_FUNCTION_RE = /\bfunction\b(\s+[\s\S]+|\s*)\([\s\S]*\)[\s\S]*\{[\s\S]*\[[\s\S]*\bnative\b[\s\S]+\bcode\b[\s\S]*\][\s\S]*\}/;
```
Comment (michaelficarra) [inline harness/nativeFunctionMatcher.js]: I think you can get away with just deleting `[\s\S]+\b\w+\b` from the original regexp here.

**Comment by michaelficarra** [comment]
LGTM otherwise. 👍

File: harness/nativeFunctionMatcher.js (hunk 1 above)
Comment (jugglinmike) [inline harness/nativeFunctionMatcher.js]: You are correct! I've done exactly that in a follow-up commit

**Comment by jugglinmike** [comment]
Thanks for the review, Michael. In addition to your suggested simplification, I've moved the documentation describing the regular expression to the harness file itself.

**Comment by michaelficarra** [comment]
Even better! :+1:

**Pull request merged**